core/src/avm2/globals/flash/net/SharedObject.as

```diff
@@ -30,6 +30,7 @@ package flash.net {
 
         // note: this is supposed to be a read-only property
         public var data: Object;
+        public var objectEncoding: uint;
```

Comment:
Hello, and thanks for contributing! `objectEncoding` is a getter/setter in Flash, could you change it here to match? Also, there should be a stub warning when calling the setter.

Reply:
Thank you for your review. I have created a getter and a setter now.